src/mumble/GlobalShortcut_win.cpp

```diff
@@ -899,3 +900,24 @@ QString GlobalShortcutWin::buttonName(const QVariant &v) {
 bool GlobalShortcutWin::canSuppress() {
 	return bHook;
 }
+
+bool GlobalShortcutWin::areScreenReadersActive() {
+	// This list contains valid executables we consider to be 'screen readers'.
+	// Todo: perhaps make this a configuration option in mumble to let users dynamically add executables.
+	const QStringList executables = {"nvda.exe", "jfw.exe"};
+	const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
+	if (snapshot != INVALID_HANDLE_VALUE) {
+		PROCESSENTRY32 p;
+		p.dwSize = sizeof(p);
+		auto ok = Process32First(hSnap, &processEntry);
```

Comment:
Sorry, I accidentally forgot to rename `hSnap` to `snapshot` after copying the code from `mumble_plugin_win32.h`.